.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,29 @@
-Please use [PX4 Discuss](http://discuss.px4.io/) or [Discord](https://discord.gg/dronecode) to align on pull requests if necessary. You can then open draft pull requests to get early feedback.
+<!--
 
-## Describe problem solved by this pull request
-A clear and concise description of the problem this proposed change will solve. Or, what it will improve.
-E.g. For this use case I ran into...
+Thank you for your contribution!
 
-## Describe your solution
-A clear and concise description of what you have implemented.
+Get early feedback through
+- Dronecode Discord: https://discord.gg/dronecode
+- PX4 Discuss: http://discuss.px4.io/
+- opening a draft pr and sharing the link
 
-## Describe possible alternatives
-A clear and concise description of alternative solutions or features you've considered.
+-->
```

Comment:
We could add something like this: (anything that reminds users to upload the ULog!! As daniel said during the dev call.

```
If you have a log, make sure to upload it to https://review.px4.io/ (it accelerates review / feedback process)
```